pkg/test/constants/constants.go

```diff
@@ -35,6 +35,10 @@ var TestNameRegexesToSkip = []string{
 	".*(streamingdataflowjobupdateparameters).*",
 	// TODO(b/267510222): re-enable calendarbudget test when test GCP org configconnector.net is allowlisted.
 	".*(calendarbudget).*",
+	// Creating Remote Control Plane Clusters is no longer supported.
+	// Need to allowlist test project to bypass this.
+	// Playbook: https://g3doc.corp.google.com/cloud/verticals/edge/g3doc/cluster_oncall_rotation.md?cl=head#creating-remote-control-plane-clusters-is-no-longer-supported
+	".*(edgecontainercluster).*",
```

Comment:
Nice you beat me to it 💯 ; thanks for adding this 🏅 

Let's not link the g3doc here for now. We can create an internal issue to allow list the test project.